README.md

```diff
@@ -77,3 +77,25 @@ Ex. For `v6.1.0` it is `v6.1.0-fh`
 
 Change the persistencecore image in `graph-node/firehose-node.yaml` file.  
 And run `kubectl apply`.
+
+## How to deploy a subgraph
```

Comment:
Just add a note here that this is a temporary setup. We plan to migrate it to auth based setup.